File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--simple-data-binding-in-vsto-add-in-project_5.cs
            this.button1 = extendedDocument.Controls.AddButton(range1, 60, 15, "1");
            this.button1.Text = "Previous";
            this.button2 = extendedDocument.Controls.AddButton(range2, 60, 15, "2");
            this.button2.Text = "Next";

            this.customerContentControl = extendedDocument.Controls.AddRichTextContentControl(
                range3, "richTextContentControl1");

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--simple-data-binding-in-vsto-add-in-project_5.vb
        Me.button1 = extendedDocument.Controls.AddButton(range1, 60, 15, "1")
        Me.button1.Text = "Previous"
        Me.button2 = extendedDocument.Controls.AddButton(range2, 60, 15, "2")
        Me.button2.Text = "Next"

        Me.customerContentControl = extendedDocument.Controls.AddRichTextContentControl(range3, _
            "richTextContentControl1")